kratos/input_output/logger_table_output.cpp

```diff
@@ -24,7 +24,7 @@
 
 namespace Kratos
 {
-	LoggerTableOutput::LoggerTableOutput(std::ostream& rOutputStream, std::vector<std::string> const& ColumnsNames) : LoggerOutput(rOutputStream), mColumnsNames(ColumnsNames) {
+	LoggerTableOutput::LoggerTableOutput(std::ostream& rOutputStream, std::vector<std::string> const& ColumnsNames) : LoggerOutput(rOutputStream), mCurrentColumnIndex(0), mColumnsNames(ColumnsNames) {
         for(auto& column_name : mColumnsNames){
             mColumnsWidth.push_back(column_name.size());
```

Comment:
@pooyan-dadvand Have you rewrite from scratch the table stream?, because in the class table_stream.h there are some tweaks to print correctly in scientific notation and so on